bundle/Core/Persistence/Legacy/Tags/Gateway/DoctrineDatabase.php

```diff
@@ -154,6 +161,13 @@ public function getChildren(int $tagId, int $offset = 0, int $limit = -1, ?array
         }
 
         $query = $this->createTagFindQuery($translations, $useAlwaysAvailable);
+
+        if ($showHidden !== null && $showHidden === false) {
```

Comment:
If `$showHidden` were not nullable, as mentioned before, you wouldn't need double check here.